lib/resty/kafka/request.lua

```diff
@@ -60,6 +60,15 @@ local function str_int64(int)
                 tonumber(band(int, 0xff)))
 end
 
+-- for easy to debug
+function mt.__tostring( self )
+    local str = ""
+    for _,v in ipairs(self._req) do
+        str = str .. tostring(v) .. " "
```

Comment:
we should push the segments to table first and then use `table.concat`.
I know this's just for debug, but it's really not a good example.